src/t_string.c

```diff
@@ -304,34 +336,35 @@ int parseExtendedStringArgumentsOrReply(client *c, int *flags, int *unit, robj *
     return C_OK;
 }
 
-/* SET key value [NX] [XX] [KEEPTTL] [GET] [EX <seconds>] [PX <milliseconds>]
- *     [EXAT <seconds-timestamp>][PXAT <milliseconds-timestamp>] */
+/* SET key value [NX | XX | IFEQ comparison-value] [KEEPTTL] [GET] [EX <seconds>]
+ *     [PX <milliseconds>] [EXAT <seconds-timestamp>][PXAT <milliseconds-timestamp>] */
```

Comment:
I see you corrected `NX | XX | IFEQ` here, but let's correct the other arguments too, to indicate which ones are mutually exclusive. (For example, we don't allow combining EX and PX with each other or with KEEPTTL.) It's not introduced in this PR, but still, I don't like comments that are wrong.

```suggestion
/* SET key value [NX | XX | IFEQ comparison-value] [GET]
 *     [EX seconds | PX milliseconds |
 *      EXAT seconds-timestamp | PXAT milliseconds-timestamp | KEEPTTL] */
```